utils/wasm-instrument/src/syscalls.rs

```diff
@@ -599,16 +610,16 @@ impl SysCallName {
 /// belongs to. See [`PtrInfo`] and [`PtrType`] for more details.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
 pub enum ParamType {
-    Size,            // i32 buffers size in memory
-    Ptr(PtrInfo),    // i32 pointer
-    Gas,             // i64 gas amount
-    MessagePosition, // i32 message position
-    Duration,        // i32 duration in blocks
-    Delay,           // i32 delay in blocks
-    Handler,         // i32 handler number
-    Alloc,           // i32 alloc pages
-    Free,            // i32 free page
-    Version,         // i32 version number of exec settings
+    Length,              // i32 buffers length
+    Ptr(PtrInfo),        // i32 pointer
+    Gas,                 // i64 gas amount
+    Offset,              // i32 offset in the input buffer (message payload)
+    DurationBlockNumber, // i32 duration in blocks
+    DelayBlockNumber,    // i32 delay in blocks
```

Comment:
```suggestion
    Duration, // i32 duration in blocks
    Delay,    // i32 delay in blocks
```
I think it's clear from the context that `delay` and `duration` are defined in the terms of blocks as this approach is used everywhere in gear

Reply:
So if we agree on that https://github.com/gear-tech/gear/pull/3541#discussion_r1410546814, I think we can resolve this too.